hack/kind-cluster.sh

```diff
@@ -40,18 +40,68 @@ nodes:
         extraArgs:
             v: "5"
 - role: worker
-  extraMounts:
-      - hostPath: /sys/fs/bpf
-        containerPath: /sys/fs/bpf
-        propagation: Bidirectional
 - role: worker
-  extraMounts:
-      - hostPath: /sys/fs/bpf
-        containerPath: /sys/fs/bpf
-        propagation: Bidirectional
 EOF
 }
 
+# install_bpf_daemonset will install the daemonset that mounts the bpf file system
+# into each kind docker container
+install_bpf_daemonset() {
+  cat <<'EOF' | kubectl apply -f -
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: bpf-mounter
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: bpf-mounter
+  template:
+    metadata:
+      labels:
+        app: bpf-mounter
+    spec:
+      hostNetwork: true
+      hostPID: true
+      tolerations:
+        - operator: Exists
+      initContainers:
+        - name: mount-bpffs
+          image: 'kindest/node:v1.25.2'
+          command:
+          - /bin/bash
+          - -c
+          - |
+            #!/bin/bash
+            if ! mount | grep -q 'bpffs on /sys/fs/bpf'; then
```

Comment:
FYI Cilium's way
```sh
  if [[ \$(/bin/mount | /bin/grep /sys/fs/bpf -c) -eq 0 ]]; then \\
    /bin/mount bpffs /sys/fs/bpf -t bpf; \\
  fi
```

Reply:
let me use taht one

Reply:
For whatever reason, I couldn't get the cilium example to work.

I'm now using:
~~~
          - /bin/bash
          - -xc
          - |
            #!/bin/bash
            if ! /bin/mount | /bin/grep -q 'bpffs on /sys/fs/bpf'; then
              /bin/mount bpffs /sys/fs/bpf -t bpf
            fi
~~~

And that yields:
~~~
$ oc logs bpf-mounter-99kvj -c mount-bpffs
+ /bin/mount
+ /bin/grep -q 'bpffs on /sys/fs/bpf'
+ /bin/mount bpffs /sys/fs/bpf -t bpf
~~~

That should be o.k., I don't know why cilium is counting instead of using `q` anyway